docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-option-feature-integration.md

```diff
@@ -0,0 +1,465 @@
+use Spryker\Zed\MerchantProductOption\Communication\Plugin\ProductOption\MerchantProductOptionGroupExpanderPlugin;
+use Spryker\Zed\MerchantProductOptionGui\Communication\Plugin\ProductOptionGui\MerchantProductOptionListTableQueryCriteriaExpanderPlugin;
+use Spryker\Zed\ProductOption\ProductOptionDependencyProvider as SprykerProductOptionDependencyProvider;
+
+class ProductOptionDependencyProvider extends SprykerProductOptionDependencyProvider
+{
+    /**
+     * @return \Spryker\Zed\ProductOptionGuiExtension\Dependency\Plugin\ProductOptionListActionViewDataExpanderPluginInterface[]
+     */
+    protected function getProductOptionListActionViewDataExpanderPlugins(): array
+    {
+        return [
+            new MerchantProductOptionListActionViewDataExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return \Spryker\Zed\ProductOptionGuiExtension\Dependency\Plugin\ProductOptionListTableQueryCriteriaExpanderPluginInterface[]
+     */
+    protected function getProductOptionListTableQueryCriteriaExpanderPlugins(): array
+    {
+        return [
+            new MerchantProductOptionListTableQueryCriteriaExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return \Spryker\Zed\ProductOptionExtension\Dependency\Plugin\ProductOptionGroupExpanderPluginInterface[]
+     */
+    protected function getProductOptionGroupExpanderPlugins(): array
+    {
+        return [
+            new MerchantProductOptionGroupExpanderPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/ProductOptionStorage/ProductOptionStorageDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\ProductOptionStorage;
+
+use Spryker\Zed\MerchantProductOptionStorage\Communication\Plugin\ProductOptionStorage\MerchantProductOptionCollectionFilterPlugin;
+use Spryker\Zed\ProductOptionStorage\ProductOptionStorageDependencyProvider as SprykerProductOptionStorageDependencyProvider;
+
+class ProductOptionStorageDependencyProvider extends SprykerProductOptionStorageDependencyProvider
+{
+    /**
+     * @return \Spryker\Zed\ProductOptionStorageExtension\Dependency\Plugin\ProductOptionCollectionFilterPluginInterface[]
+     */
+    protected function getProductOptionCollectionFilterPlugins(): array
+    {
+        return [
+            new MerchantProductOptionCollectionFilterPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Publisher/PublisherDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Publisher;
+
+use Spryker\Zed\MerchantProductOptionStorage\Communication\Plugin\Publisher\MerchantProductOption\MerchantProductOptionGroupWritePublisherPlugin;
+use Spryker\Zed\Publisher\PublisherDependencyProvider as SprykerPublisherDependencyProvider;
+
+class PublisherDependencyProvider extends SprykerPublisherDependencyProvider
+{
+    /**
+     * @return array
+     */
+    protected function getPublisherPlugins(): array
+    {
+        return array_merge(
+            // ...
```

Comment:
Examples should work out of the box, so `...` usage is not allowed, also you do not need `array_merge` here.
```
return [
      new MerchantProductOptionGroupWritePublisherPlugin(),
];
```